ci-operator/jobs/quay/quay-tests/quay-quay-tests-master-periodics.yaml

```diff
@@ -482,6 +482,8 @@ periodics:
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-quay-quay-tests-master-ocp-414-quay38-quay-e2e-tests-quay38-ocp414-lp-interop
+  prowjob_defaults:
+    tenant_id: gangway-api
   reporter_config:
     slack:
       channel: '#quay-qe'
```

Comment:
@LiZhang19817 For some reason I am unable to find this slack channel in the Red Hat internal slack. Does this channel exist? If it does could you please share a link. If not we just need to update this line to an existing slack channel.

Reply:
![Screenshot from 2023-06-06 18-50-49](https://github.com/openshift/release/assets/41153661/904bd454-c845-4506-aa92-3fa1ca9a5123)

Reply:
have you been receiving prow job reports in this channel? As far as I know the prow plugin will not update private channels. If we can direct this to a public channel then we can use this notification system to create messages in that channel like this is doing for another layered product for example. https://redhat-internal.slack.com/archives/C053E0P1H6J/p1686094558383499

Reply:
Yes, we can receive the report and monitor the testing results.

Reply:
@ascerra we've asked PROW CI team to give support to send test report to multiple slack channels. cc @dongboyan77 @chengzhang1016 
https://github.com/openshift/ci-tools/issues/3477

Reply:
The mtr-interop channel was just an example of another product teams public channel we'd want a public channel specific to Quay to send results too. The most important thing is that you get the results in a channel that you view regularly though. I wasn't aware results could be posted to private channels.

And awesome thanks for sharing this, I think we can leave it as is for now! I'll track this ticket and once if this feature is added we can add an additional channel here.